impeller/entity/entity_pass.cc

```diff
@@ -360,12 +360,20 @@ bool EntityPass::Render(ContentContext& renderer,
     auto command_buffer = renderer.GetContext()->CreateCommandBuffer();
     command_buffer->SetLabel("EntityPass Root Command Buffer");
 
-    // If the context supports blitting, blit the offscreen texture to the
-    // onscreen texture. Otherwise, draw it to the parent texture using a
-    // pipeline (slower).
+    // If the context supports blitting and the pixel formats are identical,
+    // blit the offscreen texture to the onscreen texture. Otherwise, draw it to
+    // the parent texture using a pipeline (slower).
+    auto src_format = offscreen_target.GetRenderTarget()
```

Comment:
Seems like the problem is that the pixel format for the offscreen_target does not match the root_render_target.  Can't we just make that the case?

Reply:
> Can't we just make that the case

If we dont need the offscreen to be wide gamut, we probably shouldn't take the performance hit, no?

Reply:
Oh I see, we do need it. that is what I fixed. We should discuss this next week. If essentially turning wide gamut is really a 4x performance improvement for backdrop blurs then we shouldn't have it on by default

Reply:
> Oh I see, we do need it. that is what I fixed.

No, I meant instead of making a drawing call to replace a blit making sure the blit is always possible.

Reply:
WDYM? If the texture formats don't match the blit isn't possible, that is what I'm doing. There is still a blit if they match.

Reply:
That offscreen texture is one that we create, right?  So we should be specifying that it has the same format as the final surface.  That's how this stuff was originally worked.

Reply:
CreateOffscreen has always used Capabilities.GetDefaultColorFormat(); Do we update that if the app switches into wide gamut?

If we're not doing that, then I think pretty much anything else in the entity_pass will start failing, because it doesn't seem like we've accounted for backdrop filters in with the wide gamut support?

Reply:
It's been a while since I implemented it.  I thought I made sure offscreen buffers were created to match the surface pixel format.  If that wasn't the case it seems like we'd see some nasty bugs.  At any rate, I think that is the correct fix.  Fix the setup of the offscreen buffers, not create a workaround to account for them being setup incorrectly (with added runtime cost).

Reply:
Investigated this and the offscreen format is updated to match, but the overlay surfaces introduced by platforms were not, which is the result of the mismatch.